packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -59,35 +49,6 @@ export type QueryWrapperProps = React.PropsWithChildren<{
   fileVersionColumnName?: string
 }>
 
-/** The query results, which will be undefined while initially fetching a new bundle, but will not be unloaded when fetching new pages */
-export const tableQueryDataAtom = atom<QueryResultBundle | undefined>(undefined)
```

Comment:
Removed all of these atoms that did not benefit from using atomic state, as they either

* represented server data that should be managed in react-query, or
* changed infrequently so they could be safely stored in QueryContext without performance impacts